src/test/cypress/integration/exercises/text/TextExerciseParticipation.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { artemis } from 'src/test/cypress/support/ArtemisTesting';
+import { generateUUID } from 'src/test/cypress/support/utils';
+
+// The user management object
+const users = artemis.users;
+
+// Requests
+const courseManagement = artemis.requests.courseManagement;
+
+// PageObjects
+const textEditor = artemis.pageobjects.textExercise.editor;
+
+describe('Text exercise participation', () => {
+    let course: any;
+    const exerciseTitle = 'Text exercise ' + generateUUID();
```

Comment:
default values.

Reply:
Same argumentation as above. Saving the response from the createTextExercise requests is more effort than specifying an explicit value.